share/eds/pubsub.go

```diff
@@ -0,0 +1,88 @@
+package eds
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+// pubSubTopic hardcodes the name of the EDS floodsub topic with the provided suffix.
+func pubSubTopic(suffix string) string {
+	return "eds-sub/v0.0.1/" + suffix
```

Comment:
In another PR i proposed to extract it in a const for visibility and quick access if we need to change it, but keeping in the func at the top is also fine.